secure_message/services/party_service.py

```diff
@@ -10,18 +10,31 @@
 
 
 class PartyService:
+
+    __users_cache = dict()
+    __business_details_cache = dict()
+
     @staticmethod
-    def get_business_details(ru):
-        """Retrieves the business details from the party service"""
+    def get_url(api_param, code):
+        """
+        :param api_param:  is the key configuration that represent part or the URI .
+        :param code: is the code to use in the url ( uuid or ru )
```

Comment:
Extra space between parentheses -> (uuid or ru)

Reply:
I wonder if this is the pep8 . Imean comments are not code !

Reply:
No, not PEP8. Changes requested for consistent grammar.